style-about.css

```diff
@@ -0,0 +1,525 @@
+* {
```

Comment:
- [ ] Let's try to improve the hero section of the `about` page. I think we can fix the typography bug and make sure that the size resembles the one in the template.

Yours:
![image](https://user-images.githubusercontent.com/79114233/142571930-bae0ee29-c3eb-4438-afce-a3940b35b8f5.png)

Reference:
![image](https://user-images.githubusercontent.com/79114233/142571959-095a7e21-f3ed-4b74-b9f3-265be031fe1f.png)